lighthouse-core/config/config-helpers.js

```diff
@@ -122,41 +370,25 @@ function expandAuditShorthand(audits) {
  * @param {string=} configDir
  * @return {Array<LH.Config.AuditDefn>|null}
  */
-function requireAudits(audits, configDir) {
-  const expandedAudits = expandAuditShorthand(audits);
-  if (!expandedAudits) {
+function resolveAuditsToDefns(audits, configDir) {
+  if (!audits) {
     return null;
   }
 
   const coreList = Runner.getAuditList();
-  const auditDefns = expandedAudits.map(audit => {
+  const auditDefns = audits.map(auditJson => {
+    const auditDefn = expandAuditShorthand(auditJson);
     let implementation;
-    if ('implementation' in audit) {
-      implementation = audit.implementation;
+    if ('implementation' in auditDefn) {
+      implementation = auditDefn.implementation;
     } else {
-      // See if the audit is a Lighthouse core audit.
-      const auditPathJs = `${audit.path}.js`;
```

Comment:
extracted this into `requireAudit` for structural consistency with `resolveGathererToDefn`